### ISSUE TITLE
'get-login' is no longer supported and it's not clear how to authenticate

### ISSUE THREAD

**Issue by javadba** [awslabs/ecs-nginx-reverse-proxy#5]
In the instructions Step #3 not only does not work as given 

> $ `aws ecr get-login`

The "replacement" `get-login-password` does not contain the "URI" as described in Step#4

> Tag your images and push them to the repositories. (You must put it your own repository URI from the repositoryUri value from step #3)

So I'm  uncertain how to proceed.

**Comment by ar-ds** [comment]
Yes! How can we still utilize IAM roles to authenticate to a reserved proxy ECR utilizing the provided command? 

aws ecr get-login-password --region region | docker login --username AWS --password-stdin registry.myregistry.com
